setup.sh

```diff
@@ -80,23 +106,23 @@ do
     mm=$(echo $sim_day | cut -f2 -d-)
     dd=$(echo $sim_day | cut -f3 -d-)
     if [ $REAN_STR == "ERA" ]; then
-        sed "s/SETPASSWD/nrel1234/g" $HOME_DIR/get_erai_template.csh > get_erai1.csh
-        sed "s/DD1/$dd/g" get_erai1.csh > get_erai2.csh && rm get_erai1.csh 
-        sed "s/MM1/$mm/g" get_erai2.csh > get_erai3.csh && rm get_erai2.csh 
-        sed "s/YY1/$yy/g" get_erai3.csh > get_erai.csh && rm get_erai3.csh 
-        chmod 777 get_erai.csh
-        ./get_erai.csh
-        cp get_erai.csh get_erai.csh_$CASE_STR"_"$counter
+        sed "s/SETPASSWD/$passwd/g" $TEMPLATE_DIR/get_erai_template.csh > get_erai.csh
+        sed -i "s/MY.EMAIL@asdf.com/$emailaddr/g" get_erai.csh
```

Comment:
I don't know how when I was searching for "how to use sed" this never came up. Everything I saw was saying you should save to a new file name every time... this is much better.